bundler/lib/bundler/installer/standalone.rb

```diff
@@ -16,7 +16,7 @@ def generate
         file.puts "ruby_version = RbConfig::CONFIG[\"ruby_version\"]"
         file.puts "path = File.expand_path('..', __FILE__)"
```

Comment:
Unrelated, but can we also have this:
```suggestion
        file.puts "path = File.expand_path(__dir__)"
```